light-cli/src/commands/utxo/index.ts

```diff
@@ -0,0 +1,74 @@
+import { Args, Command, Flags } from "@oclif/core";
+import {
+  CustomLoader,
+  generateSolanaTransactionURL,
+  getUser,
+} from "../../utils/utils";
+import { TOKEN_REGISTRY, User } from "@lightprotocol/zk.js";
+
+class MergeUtxosCommand extends Command {
+  static description = "Merge multiple UTXOs into a single UTXO";
+
+  static flags = {
+    latest: Flags.boolean({
+      char: "l",
+      description: "Use the latest UTXOs",
+      default: true,
+    }),
+    token: Flags.string({
+      name: "token",
+      description: "Token of the UTXOs to merge",
+      required: true,
+    }),
+  };
+
+  static args = {
+    commitments: Args.string({
+      name: "commitments",
+      description: "Commitments of the UTXOs to merge",
+      required: true,
+      multiple: true,
+    }),
+  };
+
+  protected finally(_: Error | undefined): Promise<any> {
+    process.exit();
+  }
+
+  static examples = [
+    "$ light merge-utxos --latest --token USDC 0xcommitment1 0xcommitment2 0xcommitment3",
```

Comment:
This command is meant to primarily accept inbox utxos by merging them into the main balance.

pls add a --all-inbox --asset flag which executes the command that merges all inbox utxos of a specific asset

Reply:
pls also rename the directory to merge-utxos

Reply:
added allInbox command e.g:

light merge-utxos --latest --token USDC --all